presto-jdbc/src/main/java/com/facebook/presto/jdbc/ColumnInfo.java

```diff
@@ -227,6 +233,8 @@ private static int getType(TypeSignature type)
                 return Types.DATE;
             case "decimal":
                 return Types.DECIMAL;
+            case "float":
+                return Types.REAL;
```

Comment:
That's right.
IIRC:
MySQL has float, real and double. They support non-standard precision specification in parenthesis so float and double could probably be used interchangeable (not sure, to be checked if needed). E.g. if higher precision is specified float becomes 64b representation, otherwise it's single precision.
Postgres supports _REAL_ and _DOUBLE PRECISION_. _FLOAT_ is used as a parametrized type. E.g. _FLOAT(25)_ = _DOUBLE_, _FLOAT(3)_ = _REAL_.
In Oracle there's _NUMBER_ which can be parametrized. If floating point is needed, they use _BINARY_FLOAT_ (32b) and _BINARY_DOUBLE_ (64b).
Teradata: no _REAL_, nor _FLOAT_. Only _DOUBLE_ (64b) exists.
SQL Server (as you mentioned): _REAL_ = 32b, _FLOAT_ = 64b.

So, I think there's no much consistency among databases in this matter. ;)
My understanding of standard is as I presented above but in short here:
_REAL_ = _FLOAT_
_DOUBLE_ = _FLOAT(53)_